_IBC/synternet-osmosis.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "synternet",
```

Comment:
I overlooked this before, but the chains have to be in alphabetical order (for chain_1 vs chain_2 and the file names)

Reply:
Updated.